### PR TITLE
Dropping Support for Python v3.9 and adding Python 3.12 Support in Release v24.10

### PR DESCRIPTION
**Pull request by caryr35**
This PR is Dropping Support for Python v3.9 and adding Python 3.12 Support in Release v24.10

### CODE REVIEW

**Review by bdice — APPROVED** [review]

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
cc @jameslamb @mmccarty (for awareness)

**Review by jameslamb — APPROVED** [review]
🤝

**Comment by jakirkham** [comment]
Based on offline discussion, we are going to wait a day or so for feedback

Deferring to Alex on when it makes sense to merge

File: _notices/rsn0040.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 40 # should match notice number
+notice_pin: false # set to true to pin to notice page
```
Comment (aravenel) [inline _notices/rsn0040.md:9]:
```suggestion
notice_pin: true # set to true to pin to notice page

```

**Review by aravenel — CHANGES_REQUESTED** [review]

**Review by aravenel — APPROVED** [review]

**Pull request merged**